packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-    }).create();
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('It asserts the presence of checked=', function() {
-  expectAssertion(function() {
-    runAppend(view);
-  }, /you must use `checked=/);
-});
-
-QUnit.module('{{input type=boundType}}', {
-  setup() {
-    commonSetup();
-
-    controller = {
-      inputType: 'checkbox',
-      isChecked: true
-    };
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input type=inputType checked=isChecked}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should append a checkbox', function() {
-  equal(view.$('input[type=checkbox]').length, 1, 'A single checkbox is added');
-});
-
-// Checking for the checked property is a good way to verify that the correct
-// view was used.
-QUnit.test('checkbox checked property is updated', function() {
-  equal(view.$('input').prop('checked'), true, 'the checkbox is checked');
-});
-
-QUnit.module('{{input type=\'checkbox\'}} - static values', {
-  setup() {
-    commonSetup();
-
-    controller = {
-      tab: 6,
-      name: 'hello',
-      val: false
-    };
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input type="checkbox" disabled=true tabindex=6 name="hello" checked=false}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should begin disabled if the disabled attribute is true', function() {
-  ok(view.$().is(':not(:disabled)'), 'The checkbox isn\'t disabled');
-});
-
-QUnit.test('should support the tabindex property', function() {
-  equal(view.$('input').prop('tabindex'), '6', 'the initial checkbox tabindex is set in the DOM');
-});
-
-QUnit.test('checkbox name is updated', function() {
-  equal(view.$('input').attr('name'), 'hello', 'renders checkbox with the name');
-});
-
-QUnit.test('checkbox checked property is updated', function() {
-  equal(view.$('input').prop('checked'), false, 'the checkbox isn\'t checked yet');
-});
-
-QUnit.module('{{input type=\'text\'}} - null/undefined values', {
-  setup() {
-    commonSetup();
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('placeholder attribute bound to undefined is not present', function() {
-  view = View.extend({
-    [OWNER]: owner,
-    controller: {},
-    template: compile('{{input placeholder=someThingNotThere}}')
-  }).create();
-
-  runAppend(view);
-
-  ok(!view.element.childNodes[1].hasAttribute('placeholder'), 'attribute not present');
-
-  run(null, set, view, 'controller.someThingNotThere', 'foo');
-
-  equal(view.element.childNodes[1].getAttribute('placeholder'), 'foo', 'attribute is present');
-});
-
-QUnit.test('placeholder attribute bound to null is not present', function() {
```

Comment:
moved to `@htmlbars Helpers test: {{input type='text'}}` : [`@test null values`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R375)